docs/WindowsSubsystemForLinux.md

```diff
@@ -1,60 +0,0 @@
-# Swift on the Windows Subsystem for Linux (WSL)
-
-- Note that all compiled Swift binaries are only executable within Bash on
-  Windows and are Ubuntu, not Windows, executables (WSL can natively run Ubuntu
-  executables).
-- Make sure to run all commands from `bash`, or the project won't compile.
-
-###  1. Install WSL
-Install and run the latest version of [Bash on Ubuntu on
-Windows](https://docs.microsoft.com/en-us/windows/wsl/about) installed on
-your PC.
-```
-bash
-```
-
-### 2. Install dependencies
-Install the developer dependencies needed to compile the Swift project. These
-are identical to the Ubuntu dependencies, with the addition of `make`.
-```bash
-sudo apt-get install git make cmake ninja-build clang python uuid-dev libicu-dev icu-devtools libbsd-dev libedit-dev libxml2-dev libsqlite3-dev swig libpython-dev libncurses5-dev pkg-config libblocksruntime-dev libcurl4-openssl-dev
-```
-
-### 3. Upgrade `clang`
-Install a version of `clang` with C++ 14 support; the default version of `clang`
-on WSL results in linker errors during compilation.
```

Comment:
Here

Reply:
Hahah, I bet that is due to BFD rather than gold :-).